semantic/src/site/collections/menu.overrides

```diff
@@ -1,3 +1,10 @@
 /*******************************
          Site Overrides
-*******************************/
\ No newline at end of file
+*******************************/
+
+@media only screen and (max-width: @largestMobileScreen) {
+  // For review page tabs when mobile, but probably good site wide
+  .ui.attached.tabular.menu {
+    flex-wrap: wrap;
+  }
+}
```

Comment:
Good to be able to wrap the tabs. However, the upper ones look a little weird just "floating" above the rest of the row and not attached to the container beneath.

Not sure what the solution is. It's a slight improvement if you add the border to the bottom (it only goes on left, right, top normally), but still doesn't look great. Would it be reasonable to *switch* the order so that the "active" one was always on the bottom row, or would that be annoying?

Any other suggestions?